students/jakielczyk_miroslaw/lesson_03_functions/Exponentiation.py

```diff
@@ -0,0 +1,9 @@
+# Exponentiation
+a = float(input())
+n = int(input())
+def power (a, n):
```

Comment:
there should be blank line before function definition
> Surround top-level function and class definitions with two blank lines.
Method definitions inside a class are surrounded by a single blank line.

https://www.python.org/dev/peps/pep-0008/#blank-lines